app/views/helpers/menu.php

```diff
@@ -645,10 +647,7 @@ public function displayMenu(
 
         <li>
         <?php
-        echo $this->Html->div('loader-small loader', '', array(
-            'id' => '_'.$sentenceId.'_in_process',
-            'style' => 'display:none',
-        ));
+        echo "<md-progress-circular  md-diameter='11px' style='display: none;' id='_" . $sentenceId ."_in_process'></md-progress-circular>";
```

Comment:
The loaders up there for the favorites and correctness can be removed, and you can use this `#_{sentenceId}_in process` loader instead.